platform/features/conductor/core/res/templates/time-conductor.html

```diff
@@ -77,7 +77,7 @@
                     </span>
                 </span>
 
-                <input type="submit" class="off">
+                <input type="submit" class="off hide">
```

Comment:
`display: none` breaks form submission in some browsers. See https://github.com/nasa/openmct/pull/2020#issuecomment-387827362 . Is there another approach we can take here?

Reply:
Ah, wondered why this was there, should have known. Yes, think I can set to 0 height and width or similar.

Reply:
Fixed.